### PR TITLE
build: create doctor card UI

### PR DESCRIPTION
**Pull request by raghdarizq**
The UI design for creating a doctor card incorporates various elements to present the necessary information effectively.

Image and Name and Specialization and Price and Profile Button

### CODE REVIEW

File: src/components/GridCard.tsx
Comment (YasserAmro0) [inline src/components/GridCard.tsx:1]: try to put class File to handle style

**Review by YasserAmro0 — CHANGES_REQUESTED** [review]

**Review by YasserAmro0 — APPROVED** [review]

**Review by ahmedmurtaja — APPROVED** [review]

**Review by ahmedmurtaja — CHANGES_REQUESTED** [review]
change names in constants
doctorName instead of Named

File: src/components/TherapistCard/GridCard.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea } from '@mui/material';
+import { card,button,Named } from './classes';
+import Button from '@mui/material/Button';
+export default function GridCard() {
```
Comment (LinaYahya) [inline src/components/TherapistCard/GridCard.tsx]: please change both filename and component name

File: src/components/TherapistCard/GridCard.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea } from '@mui/material';
+import { card,button,Named } from './classes';
```
Comment (LinaYahya) [inline src/components/TherapistCard/GridCard.tsx]:
```suggestion
import classes from './classes';
```

File: src/components/TherapistCard/GridCard.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import Button from '@mui/material/Button';
+export default function GridCard() {
+  return (
+    <Grid item xs={3}  sx={{marginTop:5}}>
+    <Card sx={card}>
```
Comment (LinaYahya) [inline src/components/TherapistCard/GridCard.tsx]:
```suggestion
    <Card sx={classes.card}>
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by MohannadSabaa — APPROVED** [review]

**Review by ahmedmurtaja — APPROVED** [review]

**Pull request merged**